dataflux_pytorch/benchmark/checkpointing/multinode/README.md

```diff
@@ -4,56 +4,138 @@ This benchmarking script will allow you to run and benchmark the performance of
 
 ## Getting started
 
-### Configuration
+### Setup
+ 
+#### Install requirements
+Create a Python virtual environment and activate it
+```shell
+python3 -m venv .venv
+source .venv/bin/activate
+```
 
-First ensure you are running within a virtual python enviroment, make sure gcloud config project is set to correct value. Otherwise use the following command to set it 
+Run the following commands from the root of the repo to install the packages needed by the benchmarking code
 
 ```shell
-gcloud config set project {PROJECT_ID}
+pip install -r dataflux_pytorch/benchmark/requirements.txt
+pip install .
 ```
 
-### Environment variables:
+#### Auth
 
-You will need to set the following environment variables in order for the benchmarking code to run properly. 
-1. Set the environment variables required to run the demo. These include:
-  
-  * `PROJECT`: The GCP project you are using
-  
-  * `CKPT_DIR_PATH`: The full path of the directory in which to save checkpoints, in the format `gs://<bucket>/<directory>/`
+Set up credentials for the benchmark code to be able to access your GCS bucket
 
-  * `CKPT_RESTORE_PATH`: The path to restore checkpoints from. Typically the `CKPT_DIR_PATH` + `/checkpoints/`
+```shell
+gcloud config set project PROJECT_ID
+gcloud auth application-default login
+```
 
-2. Set the optional environment variables, if desired:
-  
-  * `NUM_LAYERS`: The number of layers in the model, which affects the size of the model and therefore the size of the checkpoints. Defaults to 10.
-  
-  * `ACCELERATOR`: Set to `gpu` if running on a GPU, or `cpu` if running on a CPU (default)
-    * If running on a GPU, you also must set `PJRT_DEVICE` to `CPU`.
+## Run
+
+
+### Environment variables
+
+Set the following environment variables by updating the deployment file if deploying on a GKE cluster, or by running `export ENV_VAR=value` if running locally on your machine.
+
+1. The following environment variables must be set:
   
-  * `NUM_SAVE_CALLS`: The number of times `trainer.save_checkpoint` is called. Defaults to 3.
+    * `PROJECT`: Your GCP project.
+    
+    * `CKPT_DIR_PATH`: The path to the directory the checkpoint files will be written to.
+
+    * `CKPT_RESTORE_PATH`: The path to the directory the checkpoints will be loaded from.
```

Comment:
I think this is usually (or does it have to be) the CKPT_DIR_PATH + "/checkpoints/"?

Reply:
It must be CKPT_DIR_PATH + "/checkpoints".  Added an example.